frontend/src/metabase/query_builder/containers/QueryBuilder.unit.spec.tsx

```diff
@@ -283,4 +291,32 @@ describe("QueryBuilder", () => {
       expect(mockEvent.returnValue).not.toEqual(BEFORE_UNLOAD_UNSAVED_MESSAGE);
     });
   });
+
+  describe("renders the row count regardless of visualization type", () => {
+    const dataset = TEST_MODEL_DATASET;
+    const cards = [
+      createMockCard({ ...TEST_CARD_VISUALIZATION, display: "table" }),
+      createMockCard({ ...TEST_CARD_VISUALIZATION, display: "line" }),
+    ];
+
+    for (const card of cards) {
+      it(`renders the row count in "${card.display}" visualization`, async () => {
```

Comment:
`it.each` to it would work with IDEs

Reply:
e0f6d0eb88d222829720070d77261c2491fad2cb